lib/ansible/modules/cloud/atomic/atomic_container.py

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+ANSIBLE_METADATA = {'metadata_version': '1.0',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: atomic_container
+short_description: Manage the containers on the atomic host platform
+description:
+    - Manage the containers on the atomic host platform
+    - Allows to manage the lifecycle of a container on the atomic host platform
+version_added: "2.4"
+author: "Giuseppe Scrivano @gscrivano"
+notes:
+    - Host should support C(atomic) command
+requirements:
+    - atomic
+    - "python >= 2.6"
+options:
+    backend:
+        description:
+          - Define the backend to use for the container
+        required: True
+        choices: ["docker", "ostree"]
+        default: None
+    name:
+        description:
+          - Name of the container
+        required: True
+        default: null
+    image:
+        description:
+          - The image to use to install the container
+        required: True
+        default: null
+    rootfs:
+        description:
+          - Define the rootfs of the image
+        required: False
+        default: null
+    state:
+        description:
+          - State of the container
+        required: True
+        choices: ["latest", "absent", "latest", "rollback"]
+        default: "latest"
+    mode:
+        description:
+          - Define if it is an user or a system container
+        required: True
+        choices: ["user", "system"]
+        default: None
+    values:
+        description: >
```

Comment:
Probably don't want the ">" here.  (>" introduces a string but the "-" on the next line introduces a list entry.